src/utils/az.ts

```diff
@@ -0,0 +1,226 @@
+import {AzureAccountExtensionApi, AzureSession} from './azureAccount.api';
+import type {AzureExtensionApiProvider} from '@microsoft/vscode-azext-utils/api';
+import {SubscriptionClient, Subscription} from '@azure/arm-subscriptions';
+import {ResourceManagementClient, ResourceGroup} from '@azure/arm-resources';
+import {PagedAsyncIterableIterator} from '@azure/core-paging';
+import {
+   ContainerRegistryManagementClient,
+   Registry
+} from '@azure/arm-containerregistry';
+import {
+   ContainerRegistryClient,
+   ArtifactTagProperties
+} from '@azure/container-registry';
+import * as vscode from 'vscode';
+import {Errorable} from './errorable';
+
+export interface AzApi {
+   listSubscriptions(): Promise<Errorable<SubscriptionItem[]>>;
```

Comment:
Hi Oliver - Can we please add Unit test cases to these methods? These are the core methods which will be used across the new devx - draft integrations, at least UTs adds some assurance and make sure these methods are not changed by someone else which might break another person's functionality.